### PR TITLE
Added documentation deployment for major cloud providers

### PR DESCRIPTION
**Pull request by mnlx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,4 +18,45 @@ checked in the settings file):
 1. Duplicate the `.kotlin-template` folder and rename it to your package name.
 2. Add the package name to the `settings.gradle.kts` file, in the `includes` list.
 3. ???
-4. Profit!
\ No newline at end of file
+4. Profit!
+
+<br>
+<br>
```
Comment (NickSeagull) [inline README.md]: I think that for the better we can either remove this or add a horizontal separator `---`

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -18,4 +18,45 @@ checked in the settings file):
...
+```
+
+
+### Build container
+
+`TODO: Create example with container build`
+
+<br>
+<br>
+
```
Comment (NickSeagull) [inline README.md]:
```suggestion
```

**Review by NickSeagull — APPROVED** [review]
Awesome! Could you remove the `<br>`s please? They can interfere in the rendering process if we decide to move the docs to another platform :)

File: README.md (hunk 1 above)
Comment (mnlx) [reply README.md]: Done!

**Review by adrian-lorenzo — CHANGES_REQUESTED** [review]
To be honest, I believe that the example project should be in another repository.

1. It would make it very easy to just clone and use.
2. We would keep things simple and small.

What do you think? @javiertoledo @NickSeagull @mnlx @juanjoman

**Comment by javiertoledo** [comment]
@adrian-lorenzo I think we should remove it in a couple of iterations, but for now, I'd keep it here.

**Comment by mnlx** [comment]
I agree with Javi and we should also wait for our decision of the ideal developer experience before we make any bigger changes.

**Review by juanjoman — APPROVED** [review]

**Comment by juanjoman** [comment]
I'm moving this to our new docs repo @mnlx https://github.com/theam/ellmental-docs/pull/1

**Pull request closed without merging**